airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/FailureTrackingAirbyteMessageConsumer.java

```diff
@@ -49,7 +49,10 @@ public void start() throws Exception {
   }
 
   /**
-   * Wraps actual processing of each {@link AirbyteMessage}
+   * Business logic for handling {@link io.airbyte.protocol.models.AirbyteRecordMessage} and
```

Comment:
I know this is an abstract method, but is there anything more specific we could say here? `acceptTracked` makes it sound like it might accept some tracking?

Reply:
Thanks for pointing this out, I've updated the comment to better match that this functionality covers:
* storing of STATE messages
* serializing RECORD messages and storage within a buffer

with a note that this doesn't _always_ apply but is generally true